packages/storybook/src/stories/grid-columns.stories.tsx

```diff
@@ -222,6 +222,23 @@ export function NewColumnTypes() {
   );
 }
 
+export const FlexColumnsWithCheckbox = () => {
```

Comment:
It feels like the duplication of test cases and stories aren't necessary. Maybe we could have the tests export a component and have the storybook display it (putting tests first)

Reply:
I agree, it does look like duplication. I can remove the story, it is also a corner case, it doesn't really deserve its own story especially when it is covered by unit tests?

Reply:
I'm not sure about removing the story, I was more concerned about the duplication of code.